tests/test_config_parser.py

```diff
@@ -1,9 +1,20 @@
 import os
 import tempfile
 
-from numpy.testing import assert_raises
+from numpy.testing import assert_equal, assert_raises
 
-from fuel.config_parser import Configuration, ConfigurationError
+from fuel.config_parser import (Configuration, ConfigurationError,
+                                extra_downloader_converter)
+
+
+class TestExtraDownloaderConverter(object):
+    def test_iterable(self):
+        assert_equal(extra_downloader_converter(['a.b.c', 'd.e.f']),
+                     ['a.b.c', 'd.e.f'])
+
+    def test_str(self):
```

Comment:
Test the case where we have just one module specified as string, as that is the sort of corner case that is going to break.